tools/amd_build/patches/a_aten_src_ATen_native_cuda_Embedding.cu.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/aten/src/ATen/native/cuda/Embedding.cu b/aten/src/ATen/native/cuda/Embedding.cu
+index 67b3265b2..148ae4f6d 100644
+--- a/aten/src/ATen/native/cuda/Embedding.cu
++++ b/aten/src/ATen/native/cuda/Embedding.cu
+@@ -165,11 +165,11 @@ __global__ void renorm_kernel(
+   for (int i = tid; i < dim; i += blockDim.x) {
+     auto x = scalar_cast<accscalar_t>(weights[base_index + i]);
+     if (norm_type == 1) {
+-      v += std::abs(x);
++      v += fabs(x);
+     } else if (norm_type == 2) {
+       v += x * x;
+     } else {
+-      v += std::pow(x, norm_type);
++      v += powf(x, norm_type);
```

Comment:
Those functions should be templated on the input type (like the whole kernel is), but have hardcoded `float`